### DISCUSSION TITLE
Access Restriction for Sensitive Pages

### DISCUSSION THREAD

**Discussion by ctxinf** [casdoor/casdoor#3063] [Q&A]
Pages such as the built-in login page and `/user` pages, lack configurations for being hidden in casdoor. 
In a production environment, these pages must not be public. 
Is the only way to achieve this is by manually configuring the gateway whitelist or blocking them?

**Comment by hsluoyz** [comment]
@slow-groovin use CasWAF: https://github.com/casbin/caswaf
